src/big/big_decimal.cr

```diff
@@ -331,11 +333,7 @@ struct BigDecimal < Number
 
   # Converts to `BigInt`. Truncates anything on the right side of the decimal point.
   def to_big_i
-    if self >= 0
-      self.floor.value
-    else
-      self.ceil.value
-    end
+    (self >= 0 ? floor : ceil).value
```

Comment:
Why this change? It seems unrelated.

Reply:
@Sija Could you revert this change?

Reply:
I changed to use `trunc` since IMO it should be used in the first place instead of duplicating the logic with more surface for bugs like that one.